templates/18_delete.go.tpl

```diff
@@ -122,12 +150,20 @@ func (q {{$alias.DownSingular}}Query) DeleteAllP({{if .NoContext}}exec boil.Exec
 {{end -}}
 
 // DeleteAll deletes all matching rows.
-func (q {{$alias.DownSingular}}Query) DeleteAll({{if .NoContext}}exec boil.Executor{{else}}ctx context.Context, exec boil.ContextExecutor{{end}}) {{if .NoRowsAffected}}error{{else}}(int64, error){{end -}} {
+func (q {{$alias.DownSingular}}Query) DeleteAll({{if .NoContext}}exec boil.Executor{{else}}ctx context.Context, exec boil.ContextExecutor{{end}}{{if $soft}}, soft bool{{end}}) {{if .NoRowsAffected}}error{{else}}(int64, error){{end -}} {
```

Comment:
I think that maybe the false value here should be for soft deleting (false is the default)? So maybe we turn this into the `hardDelete` or `realDelete`?